examples/containers/containers-legends.html

```diff
@@ -7,7 +7,7 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Bottom bar</title>
   <link rel="stylesheet" href="../../packages/styles/dist/airship.css">
-  <script src="../../packages/components/dist/airship.js"></script>
+  <!-- <script src="../../packages/components/dist/airship.js"></script> -->
```

Comment:
why is this commented?

Reply:
The loader doesn't behave with static files. They need to be served.

And we don't need the components anyway in `styles` package tests.

Reply:
so its better to remove the line right?